content/en/docs/devops-user-guide/understand-and-manage-devops-projects/overview.md

```diff
@@ -6,3 +6,45 @@ linkTitle: "Overview"
+
+Nevertheless, the traditional master-slave architecture of Jenkins (i.e. multiple slaves work for a master) has the following shortcomings.
+
+- The entire CI/CD pipeline will crash once the master goes down.
+- Resources are not allocated equally as some slaves see pipeline jobs wait in queue while others remain idle.
+- Different slaves may be configured in different environments and require different coding languages. The disparity can cause inconvenience in management and maintenance. 
+
+## Understand KubeSphere DevOps
+
+KubeSphere DevOps projects support source code management tools such as GitHub, Git, and SVN. Users can build CI/CD pipelines through graphical editing panels (Jenkinsfile out of SCM) or create a Jenkinsfile-based pipeline from the code repository (Jenkinsfile in SCM).
+
+### Features
+
+The KubeSphere DevOps system provides you with the following features:
+
+- Independent DevOps projects for CI/CD pipelines with access control.
+- Out-of-box DevOps functions with no complex Jenkins configurations.
+- [Source-to-image (S2I)](../../../project-user-guide/image-builder/source-to-image/) and [Binary-to-image (B2I)](../../../project-user-guide/image-builder/binary-to-image/) for rapid delivery of images.
+- [Jenkinsfile-based pipelines](../../../devops-user-guide/how-to-use/create-a-pipeline-using-jenkinsfile/) for consistent user experience which support multiple code repositories.
+- [Graphical editing panels](../../../devops-user-guide/how-to-use/create-a-pipeline-using-graphical-editing-panel/) to create pipelines with a low learning curve.
+- A powerful tool integration mechanism such as [SonarQube](../../../devops-user-guide/how-to-integrate/sonarqube/) for code quality check.
+
+![pipeline-list](/images/docs/devops-user-guide/understand-and-manage-devops-projects/overview/pipeline-list.jpg)
+
+![sonarqube-result-detail](/images/docs/devops-user-guide/understand-and-manage-devops-projects/overview/sonarqube-result-detail.jpg)
+
+### KubeSphere CI/CD pipeline workflows
+
+A KubeSphere CI/CD pipeline runs on the back of the underlying Kubernetes Jenkins slaves. These Jenkins slaves can be dynamically scaled as they are dynamically provisioned or released based on the job status. The Jenkins master and slaves run as Pods on KubeSphere nodes. The master runs on one of the nodes with its configuration data stored in a volume. Slaves run across nodes while they may not be active all the time because they are created dynamically and deleted automatically as needed.
+
+When the Jenkins master receives a building request, it dynamically creates Jenkins slaves that run in Pods according to labels. At the same time, Jenkins slaves will be registered in the master. After slaves finish their jobs, they will be released and related Pods will be deleted as well.
+
+### Dynamically provision Jenkins slaves
```

Comment:
At present, the community uses agent instead of slave

Reply:
Fixed.